scenes/pianist.fountain

```diff
@@ -0,0 +1,73 @@
+[Terry](character) runs off to play. [Howard](character) watches for a
+moment before returning to his unit. [PENNY](character) watches the
```

Comment:
I prefer semantic line breaks, where every sentence is on its own line.
<http://rhodesmill.org/brandon/2012/one-sentence-per-line/>